semantics/executable-spec/src/Control/State/Transition/Extended.hs

```diff
@@ -53,6 +53,8 @@ module Control.State.Transition.Extended
 
     -- * Random thing
     Threshold (..),
+
+    sfor_,
```

Comment:
just out of curiosity, what was this needed for?

Reply:
Unfortunately `cabal-docspec` requires that every identifier used in the doctest is exported.

Reply:
See the end of the file:

```
-- | 'sfor_' is 'straverse_' with its arguments flipped. For a version
-- that doesn't ignore the results see 'Data.Traversable.for'.
--
-- >>> sfor_ ([1..4] :: [Int]) print    <---------------------------- use of this doctest requires `sfor_` to be exported
-- 1
-- 2
-- 3
-- 4
sfor_ :: (Foldable t, Applicative f) => t a -> (a -> f b) -> f ()
{-# INLINE sfor_ #-}
sfor_ = flip straverse_
```

Reply:
Documentation describing this restriction is found here: https://github.com/phadej/cabal-extras/blob/master/cabal-docspec/MANUAL.md

Reply:
> cabal-docspec doesn't interpret library code in ghci, but instead loads precompiled code. The effect is similar as using -fobject-code in GHCi (which is mandatory for packages with FFI, for example). The consequence is that cabal-docspec is unable to evaluate doctest examples in non-exported modules (other-modules), or which use non-exposed symbols.